.github/workflows/lint-vizro-ai.yml

```diff
@@ -64,9 +63,9 @@ jobs:
         run: |
           if [ -z "${{ steps.added-files.outputs.added_files }}" ];
           then
-            echo "No changelog fragment .md file within changelog.d was detected.  Run 'hatch run docs:changelog' to create such a fragment.";
+            echo "No changelog fragment .md file within changelog.d was detected.  Run 'hatch run changelog:add' to create such a fragment.";
```

Comment:
This stuff shouldn't be there - I think you need to pull in `main`.

Reply:
This got added yesterday when we added the vizro-ai package. It currently has an outdated linting script - that's why I just brought over the changes that you already added to `lint-vizro-core.yml` into `lint-vizro-ai.yml`, otherwise it will fail as it's trying to also look for a changelog file in vizro-ai. I already notified @maxschulz-COL and @Anna-Xiong , so they will clean up the CI workflows today :)

Reply:
Will revert changes here in favor for this PR to be merged first: https://github.com/mckinsey/vizro/pull/139